docs/open-source-project/release-notes/tizen-5-0-m2.md

```diff
@@ -0,0 +1,476 @@
+  - Early memory reclaiming (to swap) feature has been added.
+
+### Fixes
+
+- The bug that does not deliver low memory notification through event system has been fixed for headless devices.
+
+
+## System (Base)
+
+### New and changed features
+
+- Upgraded the following open source libraries:
+  - icu (60.2)
+  - ncurses (6.1)
+    - CVE-2018-10754, CVE-2017-16879, CVE-2017-13734, CVE-2017-13733, CVE-2017-13732, CVE-2017-13731, CVE-2017-13730, CVE-2017-13729, CVE-2017-13728
+  - sqlite (3.24.0)
+    - CVE-2018-8740
+  - procps-ng (3.3.15)
+    - CVE-2018-1122, CVE-2018-1123, CVE-2018-1124, CVE-2018-1125, CVE-2018-1126
+  - unzip (6.10c23)
+    - CVE-2018-1000031, CVE-2018-1000032, CVE-2018-1000033, CVE-2018-1000035
+  - json-glib (1.4.2)
+- Expanded the capi-base-utils API set: 
+  - Simple Date Format
+  - Locale Display Names
+
+
+## Application framework
+
+### New and changed features
+
+- Tizen Interface Definition Language (TIDL), rpc-port
+  - rpc-port is now available in the wearable profile.
+  - TIDL has been improved in terms of performance and stability.
+- Watchface complication framework
+  - API has been refined to support more complex UX.
+- App control API
+  - A new API for sending launch request asynchronously has been added.
+- Minicontrol API
+  - Minicontrol API is now available in the wearable profile.
+
+
+## Window System
+
+### New and changed features
+
+- Wayland
+  - The open source wayland has been upgraded to the 1.15.0 version.
+- Extended Wayland Protocols
+  - tizen_move_resize protocol has been added to support move and resize wl_surface at the same time by passing x,y,w,h geometry parameters.
+- libinput
+  - The ABS_MT_PRESSURE event type has been handled to support pressure-based input devices.
+  - The following libinput API set has been added to set udev monitor's event source and buffer size:
+    - libinput_udev_set_udev_monitor_event_source
+    - libinput_udev_set_udev_monitor_buffer_size
+- Enlightenment (Display Server)
+  - e_dbus has been added to optimize booting time. It is parallelizing D-Bus connection to remove waiting time during initialization of the enlightenment.
+  - Input device management in the Enlightenment devicemgr module has been moved to the Enlightenment core.
+  - Support for tizen_move_resize protocol has been added to make move and resize for client's window at the same time.
+  - Added the Vendor Driven HWC feature.
+    - The two HWC features are:
+    - HWC Planes
+      - Enlightenment decides the HWC policy with e_planes.
+      - Legacy HWC feature, which uses e_planes.
+    - HWC Windows
+      - The libtdm backend decides the HWC policy.
+      - Vendor Driven HWC feature, which uses e_hwc_windows.
+  - Enhanced the TCT for window system.
+    - The following test cases have been added to verify the software regressions of the enlightenment display server:
+      - D-Bus message system for server
+      - window stacking

```

Comment:
Why the test case names are in upper case?

Reply:
D-Bus is the proper none and you can see the text name on the wiki page below.

Reply:
Why the test case names are in lower case?
I am sorry it was "lower case".
I agree about D-bus, then all the test case names can be in upper case right?

Reply:
Yes.. It doesn't matter whether the test case name can be in upper cases or in lower cases.

Reply:
Yes.. It doesn't matter whether the test case name can be in upper cases or in lower cases.
They can be upper cases. I means that the first letter of the name could be in the upper case.

Reply:
@schanlim,
Thank you for your suggestion.
@wtprince,
Please make the first letter of the test cases in upper case.

Rephrase as:
- D-Bus message system for server
- Window stacking
- Various cases for normal and alpha windows
- Multi windows stacking
- Size and stacking for floating window
- Transient for window, notification type window
- Screen saver function, window rotation
- Various cases for window focus
- Input events for keygrab feature

Reply:
will rephrase as:
      - D-Bus message system for server
      - Window stacking
      - Various cases for normal and alpha windows
      - Multi windows stacking
      - Size and stacking for floating window
      - Transient for window, notification type window
      - Screen saver function, window rotation
      - Various cases for window focus
      - Input events for keygrab feature